src/ggrc-client/js/components/unified-mapper/mapper-results-item-description.js

```diff
@@ -0,0 +1,57 @@
+/*
+ Copyright (C) 2020 Google Inc.
+ Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+ */
+
+import canStache from 'can-stache';
+import canDefineMap from 'can-define/map/map';
+import canComponent from 'can-component';
+import template from './templates/mapper-results-item-description.stache';
+
+const viewModel = canDefineMap.extend({
+  instance: {
+    value: null,
+  },
+  hasNotes: {
+    get() {
+      return this.instance.notes !== undefined;
+    },
+  },
+  hideBlock: {
+    get() {
+      const hasAdmin = GGRC.access_control_roles.filter(
```

Comment:
Suggest using existing function to get roles
```suggestion
      const objectRoles = getRolesForType(instance.constructor.model_singular);
      const hasAdmin = objectRoles.find((item) => item.name === 'Admin');
      return !hasAdmin && !this.hasNotes;
```